apps/src/music/constants.ts

```diff
@@ -54,6 +54,16 @@ export const DEFAULT_PATTERN = {
 
 export const DEFAULT_PATTERN_LENGTH = 1;
 
+// For reference, events look like this:
+// events: [{src: 'sound_1', tick: 3}]
+export const DEFAULT_PATTERN_AI = {
+  kit: 'drums',
+  length: 2,
```

Comment:
nice! now that we have this length prop, we can probably remove the hardcoded `DEFAULT_PATTERN_LENGTH` and similar constants in the future